openwisp_monitoring/device/tests/test_models.py

```diff
@@ -223,6 +238,19 @@ def test_device_deleted(self):
         else:
             self.fail('metric was not deleted')
 
+    def test_bad_address_fail(self):
```

Comment:
Can you add another test for ipv6?

Reply:
@PabloCastellano, is it required? In this test, I have put the `ip-address` as some bogus string and the output message mentions `ipv4` specifically as it is the first of the `format` in the two possibilities (I am unsure though, I might be wrong). Do you mean the tests should also validate that the `ip-address` have the format as `family`. If yes, I don't think it can be done with **json-schema** so I will have to try something else.

Reply:
I think we don't need it anymore :+1: